content/en/cypherpunk/hal-finney/hal-finney-home-page.md

```diff
@@ -0,0 +1,99 @@
+
+## Vecchi saggi
+
+Questi sono alcuni saggi che ho scritto per pubblicare sulla lista Cypherpunks all'inizio e alla metà degli anni '90.
+
+### PGP
+
+Il programma di crittografia a chiave pubblica PGP di Phil Zimmermann ha suscitato un enorme interesse per la crittografia.
+
+[Truly Stealthy PGP](/cypherpunk/hal-finney/truly-stealthy-pgp)<br>
+    Per alcune applicazioni PGP può essere un punto dolente. Una variante nota come "Stealth PGP" lo rende meno visibile, ma la "furtività" non è perfetta. Questo articolo analizza ciò che sarebbe necessario per renderlo veramente furtivo.
+
+[PGP Math Library Docs](/cypherpunk/hal-finney/pgp-multi-precision-library-fuctions)<br>
+    Documentazione sul funzionamento delle funzioni matematiche della libreria matematica PGP.
+
+[PGP Web of Trust](/cypherpunk/hal-finney/pgp-web-of-trust-misconceptions)<br>
+    La "rete di fiducia" di PGP è fonte di idee sbagliate. Questo modello sarà adatto per un uso su larga scala nelle reti globali?<br>
+
+### Contante digitale
+
+[Chaum's Cash System](/cypherpunk/hal-finney/detecting-double-spending)<br>
+    Questo articolo cerca di descrivere la matematica che sta alla base del sistema Digital Cash di David Chaum et al. Come possono gli utenti onesti del sistema mantenere l'anonimato mentre gli imbroglioni che spendono due volte vengono smascherati? Questo saggio è stato ampiamente ripubblicato in rete.
+
+[Digital Cash and Privacy](/cypherpunk/hal-finney/digital-cash-and-privacy)<br>
+    Il contante digitale potrebbe svolgere un ruolo importante nella protezione della privacy in un mondo in cui sempre più transazioni avverranno per via elettronica.
+
+[Problems with Observers](/cypherpunk/hal-finney/down-with-observers)<br>
+    Le recenti proposte di David Chaum e di altri ricercatori includono la nozione di un chip "observer" che risiede nel "portafoglio" digitale e si assicura che non si verifichino doppie spese. Questo saggio critica questo approccio.
+    
+[The Beauty of Ecash](/cypherpunk/hal-finney/the-beauty-of-ecash)<br>
+    Un saggio un po' faceto sulla gioia di collezionare denaro elettronico. Ammirate la bellezza unica di ogni banconota digitale!
+    
+[Criticism of NetCash](/cypherpunk/hal-finney/first-reaction-to-medvinsky-netcash-proposal)<br>
+    Un gruppo che collabora con USC/ISI ha realizzato una proposta di contante digitale chiamata NetCash. Descrivo alcuni problemi fondamentali del loro sistema.
+
+[Blind Signature Patents](/cypherpunk/hal-finney/patent-search-on-blind-signatures)<br>
+    Il contante digitale è fortemente brevettato. Questi sono i risultati di una ricerca di brevetti sulle firme cieche che sono alla base degli algoritmi del contante digitale.
+
+### Remailer anonimi
+
+Un tempo gestivo due remailer anonimi. (Questi articoli discutono alcune questioni tecniche e sociali sollevate da questi servizi controversi.
```

Comment:
Manca `(For more information and a list of remailers look[ here](https://web.archive.org/web/20050212073109/http://www.cs.berkeley.edu/~raph/remailer-list.html).)` presente tra parentesi

Reply:
Rimuovere le parentesi in `(Questi articoli discutono alcune questioni tecniche e sociali sollevate da questi servizi controversi.`